mobile/lib/modules/asset_viewer/ui/video_player.dart

```diff
@@ -36,6 +38,9 @@ class VideoPlayerViewer extends HookConsumerWidget {
       ),
       showControls: showControls && !isMotionVideo,
       hideControlsTimer: hideControlsTimer,
+      // Always auto play motion pictures, this component isn't rendered when the motion pictures are not playing
+      // If it's a regular video, use the autoPlayVideo parameter
+      autoPlay: isMotionVideo || autoPlayVideo,
```

Comment:
As far as I can tell, when the asset viewer is showing a motion picture, at first it's just the regular photo viewer. If the user then plays the motion picture, it switches to the video viewer, but with all the controls disabled, and in that case `autoPlay` is needed so that the video player plays it.

It does seem counter intuitive, yeah 😅